Assets/SequenceSDK/Ethereum/Tests/ContractIntegrationTests.cs

```diff
@@ -29,7 +29,7 @@ public class ContractIntegrationTests
         EOAWallet wallet3 = new EOAWallet("0xabc0000000000000000000000000000000000000000000000000000000000003");
         SequenceEthClient client = new SequenceEthClient("http://localhost:8545/");
 
-        [Test]
+        [SetUp]
```

Comment:
nit (if its not a test anymore, rename so its not called testdeploy... anymore)

Reply:
Good call